crates/dyn-abi/src/type.rs

```diff
@@ -328,6 +328,29 @@ mod tests {
     use alloy_primitives::Address;
     use serde_json::json;
 
+    use hex_literal::hex;
+
+    macro_rules! encoder_test {
+        ($ty:literal, $encoded:ident) => {
```

Comment:
drier if you want

```suggestion
        ($($name:ident($ty:literal, $encoded:literal)),* $(,)?) => {
```

make it expand to

```rust
#[test]
fn $name() {
    let encoded = hex!($encoded);
    ...
]
```